circle.yml

```diff
@@ -25,7 +25,9 @@ jobs:
           command: ./gradlew build
       - run:
           command:
-            bash exec/prep-key.sh
+            bash exec/apk-gen.sh
+      - run:
+          command:
             chmod +x exec/apk-gen.sh
             ./exec/apk-gen.sh
```

Comment:
Still same as previous commit

Should be `bash exec/apk-gen.sh`

Reply:
removing statements 
chmod +x exec/apk-gen.sh
./exec/apk-gen.sh

and adding bash exec/apk-gen.sh ?

Reply:
Yes